test/errors-spec.js

```diff
@@ -0,0 +1,339 @@
+/* global beforeEach: false */
+/* global jasmine: false */
+/* global expect: false */
+/* global it: false */
+/* global describe: false */
+
+var parseTree = require("../src/parseTree");
+var Settings = require("../src/Settings");
+
+var defaultSettings = new Settings({});
+
+beforeEach(function() {
+    jasmine.addMatchers({
+        toParseWithError: function(util, customEqualityTesters) {
```

Comment:
Hmmm. I wonder whether we one day might have errors other than parse errors. If so, having the word `Parse` in that name would be a useful distinction. I guess the most correct and unambiguous form would be `toFailWithParseError`. I'll go for that.